cmd/kubeadm/app/constants/constants.go

```diff
@@ -538,6 +540,43 @@ func GetDNSIP(svcSubnet string) (net.IP, error) {
 	return dnsIP, nil
 }
 
+// GetKubernetesServiceCIDR returns the default Service CIDR for the Kubernetes internal service
+func GetKubernetesServiceCIDR(svcSubnetList string, isDualStack bool) (*net.IPNet, error) {
+	if isDualStack {
+		// The default address family for your cluster is the address family of the first
```

Comment:
@khenidak can you confirm this, please?

Reply:
`for your cluster` -> `for the cluster`
as this is not user facing.

Reply:
Done.